docs/theme.js

```diff
@@ -107,11 +107,37 @@ export default {
       color: 'white',
     },
   },
+
+  pill: {
+    primary: {
+      container: {
+        py: 1,
+        borderRadius: 3,
+        bg: 'white',
+      },
+      content: {
+        color: 'primary',
+      },
+      icon: {},
+    },
+  },
+
+  images: {
```

Comment:
instead of this nesting, this should just be accepting styles in the name of `closeIcon`.